baked_in.go

```diff
@@ -130,6 +130,7 @@ var (
 		"url":                           isURL,
 		"http_url":                      isHttpURL,
 		"uri":                           isURI,
+		"urn":                           isUrnRFC8141, // RFC 8141
```

Comment:
One requested change to match the rfc
```suggestion
		"urn_rfc8141":                           isUrnRFC8141, // RFC 8141
```